public/mobile-app/src/lib/ConnectedHomepage.svelte.test.ts

```diff
@@ -56,4 +57,80 @@ describe('/ConnectedHomepage.svelte', () => {
     const icon = container.querySelector('#notification-icon')
     expect(icon).toHaveTextContent('3')
   })
+
+  test("should display 'Ne plus recevoir de notifications' link when notifications are enabled", async () => {
+    // Given
+    window.localStorage.setItem('notifications_enabled', 'true')
+
+    // When
+    const { container } = render(ConnectedHomepage)
+    await new Promise(setTimeout) // wait for async calls
```

Comment:
Woah, this is a [hack to wait for all the async calls to complete](https://stackoverflow.com/a/72148910), never seen that one before 🤯 

Why do we suddenly need those? I can't remember needing them previously, did we change something?